extensions/adobe/experience/customerJourneyManagement/email-channel-context.schema.json

```diff
@@ -0,0 +1,58 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/customerJourneyManagement/email-channel-context",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Adobe CJM ExperienceEvent - MessageProfile Email Channel Details",
+  "description": "Email channel fields for the MessageProfile associated with the Adobe CJM Experience Event.",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+
+  "definitions": {
+    "emailChannelContext": {
+      "properties": {
+        "https://ns.adobe.com/xdm/experience/customerJourneyManagement/address": {
```

Comment:
Need to remove the "xdm" from your URI string. Namespace should be https://ns.adobe.com/experience/customerJourneyManagement/. This is my fault because I had XDM in the example I showed in another comment.